source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Custom Certificates on the Global CDN
+description: For contract customers who need to retain custom TLS certificates.
+contributors: [ari, alexfornuto]
+earlyaccess: true
+earlynote: A white glove Custom Certificate Concierge Service in now available and limited to  Legacy SSL Customers with a Pantheon contract, including Enterprise, EDU+, Pantheon One, Elite, and Reseller customers.
+---
+
+## Eligibility
+
+Access to this service is limited to existing Legacy SSL customers with a Pantheon contract.
```

Comment:
This is where the page needs to spell out who has access to the service. Suggest something like: 
> This service is limited contract customers using the Legacy HTTPS service which includes Enterprise, EDU+, Pantheon One, Elite, and Reseller.

Reply:
Describing the nature of the service should also go in this section - but would advise against terms like "white glove" for accessibility.

Reply:
Spelling it out sounds fine. What's wrong with "white glove"? 
https://www.merriam-webster.com/dictionary/white-glove